### PR TITLE
Determine the current version of Daemon through cli commands

### PR DESCRIPTION
**Pull request by anandrgitnirman**
using ldflags , latest tagged version , sha1 revision and the buld time are set at the compile time , 
these versioning details can  now easily  be sent back on the response from Daemon

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -6,5 +6,8 @@ PARENT_PATH=$(dirname $(cd $(dirname $0); pwd -P))
 
 pushd $PARENT_PATH
 mkdir -p build
-GOOS=$1 GOARCH=$2 go build -o build/snetd-$1-$2 snetd/main.go
+now=$(date +'%Y-%m-%d_%T')
```
Comment (raamb) [inline scripts/build]: How is the version incremented?

Comment (anandrgitnirman) [reply scripts/build]: version tags will be added before this script is run ( @vinthedark  and I discussed this as part of the build)

**Comment by raamb** [comment]
Please add a description to the PR on what changes have been made. Will be easy to understand going forward

File: scripts/build (hunk 1 above)
Comment (vsbogd) [reply scripts/build]: @anandrgitnirman , please update documentation to mirror new parameter of the build script

**Pull request merged**